src/cityreader/cityreader.py

```diff
@@ -14,20 +14,55 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
+
+class City():
+  def __init__(self, name, lat, lon):
+    self.name = name
+    self.lat = lat
+    self.lon = lon
+  def __str__(self):
+    return self.name
+    return self.lat
+    return self.lon
+
+  def get_name(self):
+    return self.name
+  def set_name(self, new_name):
+   self.name = new_name
+
+    
+  def get_lat(self):
+    return self.lat
+  def set_lat(self, new_lat):
+    self.lat = new_lat
+
+  def get_lon(self):
+    return self.lon
+  def set_lon(self, new_lon):
+    self.lon = new_lon
+
 cities = []
 
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
+  with open('src/cityreader/cities.csv', newline='') as csvfile:
+    reader = csv.reader(csvfile)
+    for row in reader:
```

Comment:
what you are missing here is calling "next(reader)" between line 52 and 53. The reason you do this is the top row in the csv file contains column names so you want to skip that row. The other thing which may be because of my computer but in line 51, you need only put "with open(cities.csv)" because you are already in the same folder as the csv file.